src/themes/SeaBreezeTheme/Resources/views/includes/userheader.html.twig

```diff
@@ -0,0 +1,13 @@
+<div id="topline" class="clearfix">
+    <div class="pull-left">{userwelcome|ucwords}</div>
+    <div class="pull-right">{{ showblockposition('search') }}</div>
+</div>
+<div id="header" class="clearfix">
+    <h1>{# adminpanelmenu #} <a href="{{ app.request.baseUrl }}" title="{{ {{ getModVar('ZConfig', 'slogan') }} }}">{{ {{ getModVar('ZConfig', 'sitename') }} }}</a></h1>
+    {% set topnavblock = showblockposition('topnav') %}
+    {% if topnavblock is empty %}
+    <div id="navi" class="z-clearer">
+        <ul id="nav">
+            <li class="page_item"><a href="{{ app.request.baseUrl }}" title="{gt text="Go to the site's home page"}">{{ __('Home') }}</a></li>
+            <li class="page_item"><a href="{{ path('zikulasearchmodule_user_main}', {"title":"{gt","text":"Site","to":"","your":"","account":"","panel'}\">{gt":"","Account\"}<\/a><\/li>":"","":"","<li":"","class":"page_item><a","href":"{modurl","this":"","site'}\">{gt":"","search\"}<\/a><\/li>":"","<\/ul>":"","<\/div>":"","{%":"","else":"","%}":"","{{":"","topnavblock":"","}}":"","endif":"","%":""}) }}
```

Comment:
maybe the problem is with `modurl` because see asymetric quotes in `"{{ path('zikulasearchmodule_user_main}'`